src/v/redpanda/application.cc

```diff
@@ -2301,6 +2316,7 @@ void application::wire_up_and_start(::stop_signal& app_signal, bool test_mode) {
     construct_service(_as).get();
 
     // Bootstrap services.
+    wire_up_and_start_crypto_services();
```

Comment:
question: Is there a rationale for not putting this in `initialize`?

Reply:
yes, I wanted `check_environment()` to happen first to verify that we are operating in a FIPS operating environment (check that `/proc/sys/crypto/fips_enabled` is 1) and I wanted it to happen after the crash loop detection in case we enter a crash loop due to some issue with initialization of the OpenSSL library.

Reply:
But I can definitely be convinced otherwise and would be more than happy to move it :)

Reply:
> But I can definitely be convinced otherwise and would be more than happy to move it :)

SGTM